static/app/views/alerts/rules/row.tsx

```diff
@@ -97,37 +97,51 @@ class RuleListRow extends React.Component<Props, State> {
     const activeIncident = this.activeIncident();
     const criticalTrigger = rule?.triggers.find(({label}) => label === 'critical');
     const warningTrigger = rule?.triggers.find(({label}) => label === 'warning');
+    const resolvedTrigger = rule?.resolveThreshold;
     const trigger =
       activeIncident && rule.latestIncident?.status === IncidentStatus.CRITICAL
         ? criticalTrigger
         : warningTrigger ?? criticalTrigger;
 
     let iconColor: Color = 'green300';
+    let iconDirection;
+    let thresholdTypeText =
+      activeIncident && rule.thresholdType === AlertRuleThresholdType.ABOVE
+        ? t('Above')
+        : t('Below');
+
     if (activeIncident) {
       iconColor =
         trigger?.label === 'critical'
           ? 'red300'
           : trigger?.label === 'warning'
           ? 'yellow300'
           : 'green300';
+      iconDirection = rule.thresholdType === AlertRuleThresholdType.ABOVE ? 'up' : 'down';
+    } else {
+      if (!rule?.latestIncident) {
+        // If there's no latest incident, use the Resolved threshold type, which is opposite of Critical
+        iconColor =
+          rule.thresholdType === AlertRuleThresholdType.ABOVE ? 'green300' : 'red300';
+        iconDirection =
+          rule.thresholdType === AlertRuleThresholdType.ABOVE ? 'down' : 'up';
+        thresholdTypeText =
+          rule.thresholdType === AlertRuleThresholdType.ABOVE ? t('Below') : t('Above');
+      }
     }
 
-    const thresholdTypeText =
-      activeIncident && rule.thresholdType === AlertRuleThresholdType.ABOVE
-        ? t('Above')
-        : t('Below');
-
     return (
       <FlexCenter>
-        <IconArrow
-          color={iconColor}
-          direction={
-            activeIncident && rule.thresholdType === AlertRuleThresholdType.ABOVE
-              ? 'up'
-              : 'down'
-          }
-        />
-        <TriggerText>{`${thresholdTypeText} ${trigger?.alertThreshold?.toLocaleString()}`}</TriggerText>
+        <StatusWrapper>
```

Comment:
FlexCenter has the same css rules is status wrapper necessary?

Reply:
oops, forgot to remove this from a previous solution